substrate/frame/revive/src/tests/sol/system.rs

```diff
@@ -40,8 +40,9 @@ fn keccak_256_works(fixture_type: FixtureType) {
 	let (code, _) = compile_module_with_type("System", fixture_type).unwrap();
 	ExtBuilder::default().build().execute_with(|| {
 		let _ = <Test as Config>::Currency::set_balance(&ALICE, 100_000_000_000);
-		let Contract { addr, .. } =
-			builder::bare_instantiate(Code::Upload(code)).build_and_unwrap_contract();
+		let Contract { addr, .. } = builder::bare_instantiate(Code::Upload(code))
+			.data(SystemFixture::constructorCall { panic: false }.abi_encode())
+			.build_and_unwrap_contract();
```

Comment:
I believe this would pass on master if we were calling eth_instantiate(code ++ data)